src/apps/frontend/pages/authentication/forgot-password/forgot-password-form.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { FormControl, Input } from '../../../components';
+import Button, { ButtonType } from '../../../components/button/button-primary';
+import constants from '../../../constants/routes';
+import { AsyncError } from '../../../types';
+
+import BackButton from './back-button';
+import useForgotPasswordForm from './forgot-password-form.hook';
+
+interface ForgotPasswordFormProps {
+  onError: (error: AsyncError) => void;
+  onSuccess: (username: string) => void;
+}
+
+const ForgotPasswordForm: React.FC<ForgotPasswordFormProps> = ({
+  onError,
+  onSuccess,
+}) => {
+  const navigate = useNavigate();
+  const isLoginLoading = false;
+  const { formik } = useForgotPasswordForm({ onError, onSuccess });
+  return (
+    <>
+      <BackButton onClick={() => navigate(constants.LOGIN)} />
+      <h2 className="mb-5 text-2xl font-bold text-black dark:text-white sm:text-title-xl2">
```

Comment:
Can we create `H2` component? I see similar css applied on H2 multiple places. The component should not have `mb5` as the layout should control spacing around